simulation_mode: true

interaction_mode: 1
# LOCAL_ONLY = 1

number_of_arduinos: 3
rs485_device_name: /dev/ttyUSB0  

max_number_of_sensors_per_arduino: 3
max_number_of_lights_per_arduino: 4

light_nodes_size:
  x: 0.1
  y: 0.1
  z: 0.1
light_nodes_color: [0.9, 0.9, 0.9, 1.0] # r,g,b,a

nodes_size:
  x: 0.07
  y: 0.07
  z: 0.07
nodes_color: [0.8, 0.8, 0.8, 1.0] # r,g,b,a

beams_size:
  x: 0.03
  y: 0.03
  z: 1.0
beams_color: [0.4, 0.4, 0.4, 1.0] # r,g,b,a

light_beams_size:
  x: 0.08
  y: 0.08
  z: 0.8
light_beams_color: [0.8, 0.4, 0.4, 1.0] # r,g,b,a

sensors_size:
  x: 0.04
  y: 0.04
  z: 0.9
sensors_color: [0.2, 0.2, 0.9, 0.9] # r,g,b,a

sensors:
  - nodes: [0, 1]
  - nodes: [1, 2]
  - nodes: [4, 6]
  - nodes: [1, 3]

light_beams:
  - nodes: [2, 6]
  - nodes: [3, 7]
  - nodes: [4, 5]
  - nodes: [5, 6]

beams:
  - nodes: [0, 1]
  - nodes: [1, 2]
  - nodes: [1, 3]
  - nodes: [2, 3]
  - nodes: [3, 0]
  - nodes: [4, 5]
  - nodes: [4, 6]
  - nodes: [5, 6]
  - nodes: [6, 7]
  - nodes: [7, 0]
  - nodes: [0, 4]
  - nodes: [1, 5]
  - nodes: [2, 6]
  - nodes: [3, 7]

light_nodes: [0, 2, 5]

light_node_connections: [0, 1, 2]
light_beam_connections: [0, 0, 1, 2]
sensor_connections: [0, 0, 1, 2]

node_positions:
  - position: [0.0, 0.0, 0.0]
  - position: [1.0, 0.0, 0.0]
  - position: [1.0, 1.0, 0.0]
  - position: [0.0, 1.0, 0.0]
  - position: [0.0, 0.0, 1.0]
  - position: [1.0, 0.0, 1.0]
  - position: [1.0, 1.0, 1.0]
  - position: [0.0, 1.0, 1.0]


number_of_simulated_objects: 20
simulated_object_name: ball
simualtion_sensor_object_threshold: 0.5
